### ISSUE TITLE
Local bubble timeline, like Akkoma

### ISSUE THREAD

**Issue by greyidol** [CatCatNya/catstodon#6]
## Pitch

Akkoma has a really neat feature called the "local bubble timeline", where the admins pick a list of instances that represent the community of the instance, and a subset of the federated timeline with those instances' posts will appear in the bubble.

## Motivation

This feature is a game changer because it helps users break out of their own instance without being overwhelmed by irrelevant posts on the federated timeline. It's like the local timeline but less of a silo.

**Comment by kescherCode** [comment]
Great suggestion. This could even be a glitch-soc feature, imo.

**Comment by kescherCode** [comment]
Indeed, a similar request has been opened on glitch-soc: https://github.com/glitch-soc/mastodon/issues/2553
